src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
+import Checkbox from './Checkbox';
+
+describe('<Checkbox />', () => {

```

Comment:
Can wait until next tri, but it would be good to have some more extensive tests here e.g., selecting and deselecting